aospy/data_loader.py

```diff
@@ -10,6 +10,26 @@
 from .utils import times, io
 
 
+def preprocess_and_rename_grid_attrs(func):
```

Comment:
Convention is to start w/ single line summary.  You don't need to specify the 'grid_atts_to_aospy_names` name explicitly, since that takes a lot of space